docs/source/custom_task.rst

```diff
@@ -4,26 +4,39 @@ Tutorial: Creating a Custom Task
 In this tutorial we will go over the process of creating a custom task,
 along with a custom data module.
 
+1 . Imports
+-----------
+
+
 .. testcode:: python
 
-    import flash
+    from typing import Any, List, Tuple
 
+    import numpy as np
     import torch
-    from torch.utils.data import TensorDataset, DataLoader
-    from torch import nn
+    from pytorch_lightning import seed_everything
     from sklearn import datasets
     from sklearn.model_selection import train_test_split
+    from torch import nn
+
+    import flash
+    from flash.data.auto_dataset import AutoDataset
+    from flash.data.process import Postprocess, Preprocess
+
+    seed_everything(42)
 
-The Task: Linear regression
----------------------------
+
+2 . The Task: Linear regression
```

Comment:
I think data should go before task.

Reply:
Steps to create a task

1. DataModule
2. PreProcess
3. Post Process
4. Registry 
5. Task
6. Example
7. Test (This is important since people need to know how to test to merge)

Reply:
Done ! Feel free to re-organize this when you have time.